### PR TITLE
Teltonika Router API

### PR DESCRIPTION
**Pull request by LukasBaecker**
This PR adds the functionality of calling the Teltonika router's API from the Field Friend UI. 
In #93 we added a checkbox for (de)activating image uploads, but would now like to add the possibility of (de)activating the usage of SIM-cards plugged into the router which might fall-back to these if the Wifi is low. 
More functionalities using the Teltonika API might follow.

Todos
- [x] connect to Teltonika API
- [x] adding the Teltonika to a real system setup
- [x] testing on a real system

### CODE REVIEW

**Comment by rodja** [comment]
@LukasBaecker what is the src folder for? Accidentally added?

**Comment by LukasBaecker** [comment]
> @LukasBaecker what is the src folder for? Accidentally added?

I have cleaned that up.

File: field_friend/interface/components/development.py [hunk 1]
```diff
@@ -28,5 +28,18 @@ def development(system: 'System') -> None:
                             rosys.simulation_ui()
                     hardware_control(system.field_friend, system.automator, system.puncher)
                     status_dev_page(system.field_friend, system)
-    with ui.row().style('width: calc(100vw - 2rem); flex-wrap: nowrap;'):        
+                    with ui.card().style('background-color: #3E63A6; color: white;'):
```
Comment (pascalzauberzeug) [inline field_friend/interface/components/development.py:31]: You could move this card as its own class to `field_friend/hardware/teltonika_router.py`

Comment (pascalzauberzeug) [reply field_friend/interface/components/development.py:31]: Oh, I wanted to mark the whole card block

Comment (LukasBaecker) [reply field_friend/interface/components/development.py:31]: No worries, these buttons are only for testing purpose and I will display the information on the UI in a smoother way later.

**Comment by LukasBaecker** [comment]
We realigned the goal of this PR to only read the current connections from the Teltonika API and not setting the conncetion from the Feldfreund UI for now. 
The system looks for all connections that are 'up' and sets the one with the highest priority to the current connection (ether > wifi > mobile).

**Comment by LukasBaecker** [comment]
The latest change uses the API endpoint "failover/status" which is a lot smoother then the one before. Currently request timeouts appear which need to be discussed. On the one hand the change of the current connection (e.g. from cable to wifi) is still performant and with the new request also more robust but still there are some request timeouts.

**Comment by LukasBaecker** [comment]
> We realigned the goal of this PR to only read the current connections from the Teltonika API and not setting the conncetion from the Feldfreund UI for now. The system looks for all connections that are 'up' and sets the one with the highest priority to the current connection (ether > wifi > mobile).

I increased the request timeout to avoid the error for requests up to 20 secs.

**Review by LukasBaecker — COMMENTED** [review]
reviewed with @angelom93

**Review by angelom93 — APPROVED** [review]
Looks good!

**Pull request merged**